site/src/main/paradox/Joins.md

```diff
@@ -0,0 +1,221 @@
+# Joins
+
+Scio provides a full suite of join functionality and a few extras that solve tricky edge-cases in large-scale data processing.
+
+All joins operate over `SCollection`s containing 2-tuples, where the first tuple item is considered the _key_ and the second the _value_.
+The order in which `SCollection`s are joined matters; larger datasets should be further to the left.
+For example in `a.join(b)`, `a` should be the larger of the two datasets and by convention `a` is called the left-hand-side or _LHS_, while `b` is the right-hand-side or _RHS_.
+
+## Union
+
+Unions are a special case where two `SCollection`s of the same type are to be combined into one.
+
+@scaladoc[`union`](com.spotify.scio.values.SCollection#union(that:com.spotify.scio.values.SCollection[T]):com.spotify.scio.values.SCollection[T])
+@scaladoc[`unionAll`](com.spotify.scio.ScioContext#unionAll[T](scs:=%3EIterable[com.spotify.scio.values.SCollection[T]])(implicitevidence$6:com.spotify.scio.coders.Coder[T]):com.spotify.scio.values.SCollection[T])
```

Comment:
Not sure if this should be in under the  section

Reply:
🤷 it's a kind of non-join join, felt better than putting it in the core behavior section, but not committed to that decision. WDYT @clairemcginty